generic/tests/netperf.py

```diff
@@ -482,6 +482,7 @@ def count_interrupt(name):
         sum = 0
         intr = []
         stat = ssh_cmd(server_ctl, "cat /proc/interrupts |grep %s" % name)
+        stat = stat.strip().split("\n")[-1]
```

Comment:
Update the code as you suggest, and I have tested with none-mq and mq guest, it's all pass. 

Thanks, wenli